src/control/lib/daos/hlc.go

```diff
@@ -21,7 +22,7 @@ import (
 
 const (
 	// ZeroHLCDate is the date of the zero HLC, as
-	// defined by CRT_HLC_START_SEC in src/cart/crt_hlc.c
+	// defined by CRT_HLC_START_SEC in src/cart/d_hlc.c
```

Comment:
```suggestion
	// defined by D_HLC_START_SEC in src/gurt/d_hlc.c
```

Reply:
changed to src/gurt/hlc.c as the file got renamed too from @liw feedback